lib/usd/translators/shading/usdUVTextureReader.cpp

```diff
@@ -311,25 +312,36 @@ bool PxrMayaUsdUVTexture_Reader::Read(UsdMayaPrimReaderContext* context)
         }
     }
 
-    // Wrap U
-    usdInput = shaderSchema.GetInput(_tokens->wrapS);
-    mayaAttr = uvDepFn.findPlug(_tokens->wrapU.GetText(), true, &status);
-    if (usdInput && status == MS::kSuccess) {
-        if (usdInput.Get(&val) && val.IsHolding<TfToken>()) {
-            TfToken wrapS = val.UncheckedGet<TfToken>();
-            val = wrapS == _tokens->repeat;
-            UsdMayaReadUtil::SetMayaAttr(mayaAttr, val);
-        }
-    }
-
-    // Wrap V
-    usdInput = shaderSchema.GetInput(_tokens->wrapT);
-    mayaAttr = uvDepFn.findPlug(_tokens->wrapV.GetText(), true, &status);
-    if (usdInput && status == MS::kSuccess) {
-        if (usdInput.Get(&val) && val.IsHolding<TfToken>()) {
-            TfToken wrapT = val.UncheckedGet<TfToken>();
-            val = wrapT == _tokens->repeat;
-            UsdMayaReadUtil::SetMayaAttr(mayaAttr, val);
+    // Wrap U/V
+    for (auto wrapMirrorTriple :
```

Comment:
We are still having trouble with windows nodes, but in this case PF actually found something real.

```S:\jenkins\workspace\ecg-mayausd-branch-preflight-2020-windows\ecg-maya-usd\maya-usd\lib\usd\translators\shading\usdUVTextureReader.cpp(316): error C4576: a parenthesized type followed by an initializer list is a non-standard explicit type conversion syntax```

Reply:
Hmm, sorry about that. I guess the initializer syntax I used was not strict c++11, but a gcc extension:

```
(const TfToken[2][3]){ ... }
```

I just ended up declaring a variable to hold the array.  Alternatively, I could apparently have declared a type alias, since it seems that

```
T{ ... }
```

is valid, but only if `T` is a "simple-type-specifier".  So I could have done:

```
using Token23 = const TfToken[2][3];
for (auto wrapMirrorTriple : Token23{ ... }) {
```

...but I decided at that point it was just simpler to make the variable...

(Thanks to this post for the info about simple-type-specifiers: https://stackoverflow.com/a/24390563/920545)